docs/integrations/engines/trino.md

```diff
@@ -60,6 +60,57 @@ hive.metastore.uri=thrift://example.net:9083
 delta.hive-catalog-name=datalake_delta # example catalog name, can be any valid string
 ```
 
+#### AWS Glue
+
+[AWS Glue](https://aws.amazon.com/glue/) provides an implementation of the Hive metastore catalog.
+
+Your Trino project's physical data objects are stored in a specific location, such as an [AWS S3](https://aws.amazon.com/s3/) bucket. Hive provides a default location, which you can override in its configuration file.
+
+Set the default location for your project's tables in the Hive catalog configuration's [`hive.metastore.glue.default-warehouse-dir` parameter](https://trino.io/docs/current/object-storage/metastores.html#aws-glue-catalog-configuration-properties).
+
+For example:
+
+```linenums="1"
+hive.metastore=glue
+hive.metastore.glue.default-warehouse-dir=s3://my-bucket/
+```
+
+If a model should not use the default location, you may specify its location in the model DDL's [`physical_properties` dictionary](../../concepts/models/overview.md#physical_properties). This value will be passed to the `LOCATION` property in the model's `CREATE` statements.
+
+For example:
+
+``` sql linenums="1"
+MODEL (
+  name my_schema.my_model,
+  physical_properties (
+    location = 's3://my-bucket/'
```

Comment:
One problem I found setting this at the model level is that it the path doesn't automatically get adjusted for each snapshot. So you have the situation where SQLMesh creates multiple snapshots for the model in the same location and they clobber each other

Reply:
That is definitely a problem! Would it make sense to pull that out into a mixin that we use in both Athena and Trino, or is it too specific?

Reply:
I think it could work as a mixin, it would essentially need to detect `s3_base_location` and rewrite it to `location` (injecting the snapshot table name) before handing the properties off to the adapter for further processing.

Perhaps a follow-up PR once the Athena one has been reviewed?